CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribution Guidelines
+
+This document aims to provide guidelines for merging changesets into the sparrow master branch.
+
+For the purpose of this document, we classify changesets/pull requests as:
+
+* big: major changes to core code
+* non-trivial: significant changes, fixes and improvements. Especially if they change core modules.
+* trivial: minor bugfixes, small changes to auxiliary modules, changes to documentation, and patches which change only test code.
+
+## Writing & Testing Code:
+
+Write type specifications and function signatures because they're remarkably helpful when it comes to reading the source.
```

Comment:
Write type specification and function signatures because they are remarkably helpful when it comes to reading the source.